### DISCUSSION TITLE
CodePilot Support in CodeServer?

### DISCUSSION THREAD

**Discussion by v10o** [coder/code-server#5687] [Q&A] [answered]
I am trying to install codePilot in CodeServer but I cannot find any extension. If anyone solved this issue, please provide me with the solution. Thanks

**Comment by v10o** [comment]
Update:
Manually Installed CodePilot extension using vsix file, logged in to github. After that getting this err 

""
GitHub Copilot requires VS Code 1.57+. Please update your VS Code

""

Note: I have latest version of the codeserver

**Comment by jsjoeio** [comment, marked as answer]
There are some previous discussions which might have solutions! Here's one of them: https://github.com/coder/code-server/discussions/4363
